applications/ConvectionDiffusionApplication/custom_elements/symbolic_d_convection_diffusion_explicit.h

```diff
@@ -0,0 +1,317 @@
+
+    void FinalizeSolutionStep(
+        const ProcessInfo& rCurrentProcessInfo) override;
+
+    void Calculate(
+        const Variable<double>& rVariable,
+        double& Output,
+        const ProcessInfo& rCurrentProcessInfo) override;
+
+    void CalculateOrthogonalSubgridScaleSystem(
+        VectorType& rRightHandSideVector,
+        const ProcessInfo& rCurrentProcessInfo);
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+    int Check(const ProcessInfo &rCurrentProcessInfo) override;
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    /// Turn back information as a string.
+    std::string Info() const override
+    {
+        return "SymbolicDConvectionDiffusionExplicitElement #";
+    }
+
+    /// Print information about this object.
+    void PrintInfo(std::ostream& rOStream) const override
+    {
+        rOStream << Info() << this->Id();
+    }
+
+    ///@}
+
+protected:
+
+    ///@name Protected member Variables
+    ///@{
+
+    VectorType mUnknownSubScale;
+
+    ///@}
+    ///@name Protected Operators
+    ///@{
+
+
+    ///@}
+    ///@name Protected Operations
+    ///@{
+
+    void CalculateLocalSystemInternal(
+        ElementVariables& rVariables,
+        MatrixType& rLeftHandSideMatrix,
+        VectorType& rRightHandSideVector);
+
+    void CalculateOrthogonalSubgridScaleSystemInternal(
+        ElementVariables& rVariables,
+        VectorType& rRightHandSideVector);
+
+    void UpdateUnknownSubgridScaleGaussPoint(
+        ElementVariables& rVariables,
+        unsigned int g);
+
+    void CalculateTau(
+        ElementVariables& rVariables);
+
+    ///@}
+    ///@name Protected  Access
+    ///@{
+
+
+    ///@}
+    ///@name Protected Inquiry
+    ///@{
+
+    ///@}
+    ///@name Protected LifeCycle
+    ///@{
+
+    // Protected default constructor necessary for serialization
+    SymbolicDConvectionDiffusionExplicit() : SymbolicQSConvectionDiffusionExplicit<TDim,TNumNodes>()
+    {
+    }
+
+    ///@}
+
+private:
+
+    ///@name Static Member Variables
+    ///@{
+
+    ///@}
+    ///@name Member Variables
+    ///@{
+
+    ///@}
+    ///@name Serialization
+    ///@{
+
+    friend class Serializer;
+
+    void save(Serializer& rSerializer) const override
+    {
+        KRATOS_SERIALIZE_SAVE_BASE_CLASS(rSerializer, Element);
+    }
+
+    void load(Serializer& rSerializer) override
+    {
+        KRATOS_SERIALIZE_LOAD_BASE_CLASS(rSerializer, Element);
+    }
+
+    ///@}
+    ///@name Private Operators
+    ///@{
+
+
+    ///@}
+    ///@name Private Operations
+    ///@{
+
+
+    ///@}
+    ///@name Private  Access
+    ///@{
+
+
+    ///@}
+    ///@name Private Inquiry
+    ///@{
+
+
+    ///@}
+    ///@name Un accessible methods
+    ///@{
+
+    /// Assignment operator.
+    SymbolicDConvectionDiffusionExplicit& operator=(SymbolicDConvectionDiffusionExplicit const& rOther);
+
+    /// Copy constructor.
+    SymbolicDConvectionDiffusionExplicit(SymbolicDConvectionDiffusionExplicit const& rOther);
```

Comment:
I'd move these to the operators section of the Doxygen and use the `=delete` keyword to preclude their usage.

Reply:
Done both things.